src/index.js

```diff
@@ -0,0 +1,40 @@
+import domNodeCreator from './domNodeCreator';
+
+function constructURL(location, unit) {
+  return `https://api.openweathermap.org/data/2.5/weather?q=${location}&appid=786b74dc8fce162462303a21c3e890c0&units=${unit}`;
+}
+
+const result = document.getElementById('result');
+
+function renderResult(temp, location, icon, weatherDesc, unit) {
+  result.innerHTML = '';
+  const imgSrc = `http://openweathermap.org/img/wn/${icon}@4x.png`;
+  const locationResultEle = domNodeCreator('p', { class: 'location' }, location.toUpperCase());
+  const weatherIconEle = domNodeCreator('img', { src: imgSrc, alt: 'weather icon', class: 'weather-icon' });
+  const weatherDescEle = domNodeCreator('p', { class: 'weather-desc' }, weatherDesc);
+  const tempResultEle = domNodeCreator('p', { class: 'temp' }, temp);
+  const unitEle = domNodeCreator('sup', { class: 'unit' }, `${unit === 'metric' ? '°C' : '°F'}`);
+  result.appendChild(locationResultEle);
+  result.appendChild(weatherIconEle);
+  result.appendChild(weatherDescEle);
+  tempResultEle.appendChild(unitEle);
+  result.appendChild(tempResultEle);
+}
+
```

Comment:
- [ ] Kindly keep the dom-manipulation tasks separate from the application logic. This will allow you to create different files for different tasks in your application which is always good practice. Kindly since you have already learned about modules in javascript, use that power to separate your application into several modules. ⬇️ 

![dom_logic_separation_requirement](https://user-images.githubusercontent.com/32361963/113996704-174aea00-9860-11eb-8c2d-c5a971c83b4f.png)